### ISSUE TITLE
Remove eCommerce

### ISSUE THREAD

**Issue by doolse** [equella/Equella#41]
We know nobody was using it as it was behind a flag in the license key. 
Being on by default now, it'll be adding a large number of settings in the Admin Console and Settings page which won't make sense to anyone.

**Comment by ddelblanco** [comment]
In this case I undestand that i need to remove ALL the code related with this, isn't it?
All the  com.tle.*.payment.*  and the references to them?

**Comment by bjagg** [comment]
doolse said he would send the folder to remove -- think it's simply the entire folder

**Comment by ddelblanco** [comment]
I've created a PR with this change. 
https://github.com/equella/Equella/pull/48

There were several plugins.
I've removed these from the reports and the admin console
I've removed them from the launch files (server and admin console ones too)... and that has been really tedious.... 

I've tested. It builds, starts and nothing seems to appear in the administrative menu about the payments, stores or prices. Anyway, I'd appreciate some extra eyes looking at it.
